service-api/app/src/App/src/Service/Lpa/ValidateOlderLpaRequirements.php

```
@@ -54,7 +54,17 @@ public function __invoke(array $lpa): bool
             );
             return false;
         }
+        return true;
```

I think it might be better to make these functions void and throw your exceptions inside here rather than return a bool and then throw the exception in the calling class

---

OK